app/forms/ineligible_gross_income_form.rb

```diff
@@ -0,0 +1,13 @@
+class IneligibleGrossIncomeForm
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+  include SessionPersistable
+
+  SELECTION = :early_eligibility_selection.to_s
+
+  ATTRIBUTES = [:early_eligibility_selection].freeze
+  VALID_OPTIONS = { gross: "gross", continue: "continue_check" }.freeze
```

Comment:
can we change gross an income to something more descriptive at first glance:
early_gross_exit and early_eligibility_continue seem like good options, that allow us to extend for disposable an cpaital